sql/catalyst/src/main/scala/org/apache/spark/sql/internal/SQLConf.scala

```diff
@@ -407,6 +407,17 @@ object SQLConf {
       .booleanConf
       .createWithDefault(true)
 
+  val REQUIRE_ALL_CLUSTER_KEYS_FOR_HASH_PARTITION =
```

Comment:
We should mention the ordering is also required if it is also in the condition.

Reply:
@viirya - sure, will change later once we all reach the consensus in comments.